docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -233,25 +233,48 @@ these are related to V&V. For more general information and best practices on V&V
+      - `Continous risk standard deviation and visualization <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_example_continuous_risks.ipynb>`_ 
+      - `Risk correlation <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_correlation.ipynb>`_ 
   * - V&V for Events with Multiple Risk Factors
     - Stratifying event rates by many risk factors might not be computationally feasible; you can verify risk effects by calculating the event rate at the simulant level.
-    - 
+    - `Multiple risks impacting CVD <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Interactive_RR_GregGraphs.ipynb>`_ 
   * - V&V for Relative Risks based on Continuous Risks 
     - For continuous risks with risk effects, simulant level data is needed to validate risk and outcome rates. 
-    - 
+    - `Same example as prior row <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Interactive_RR_GregGraphs.ipynb>`_ 
   * - Check for Simulant Level Continuity 
     - Can check that simulant values which are not meant to change, remain constant over time (example: propensities)
-    - 
+    - `Testing propensities drift <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Old_VV_unresolved/Interactive_Sim_Tests_06.18.2023_testing_propensity.ipynb>`_
   * - Debugging 
     - This is very general, but simulant level data can be helpful in finding potential issues. Some examples include: propensity drift over time or finding problematic outliers. You can also "remove" parts of the sim to see where a problem might be. 
-    - 
+    - Examples: 
+
+      - `Finding common random number error <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/validation/child/interactive_simulations/20221003%20Common%20random%20numbers%20investigation.ipynb>`_. 
+      - The `propensity drift notebook above <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Old_VV_unresolved/Interactive_Sim_Tests_06.18.2023_testing_propensity.ipynb>`_ was also an effort at debugging using the interactive sim. 
   * - Primary Output Graphs 
     - Creating visualizations when individual data is needed - such as simulant interactions with healthcare or continuous risk factor spreads over time. 
-    - 
+    - Examples: 
+
+      - `Simulant level hemoglobin changes over time <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/validation/maternal/interactive_simulations/Hemoglobin%20trajectory%20plots%2020220616.ipynb>`_ 
+      - `Simulant healthcare interactions <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Single_Simulant_Graph_Lifestyle.ipynb>`_  
+      - `Sankey diagrams <https://github.com/ihmeuw/vivarium_research_multiple_myeloma/tree/8ca7c6d23354ffb08f532d163990f18745f4c80a/verification/interactive_simulations/sankey_diagrams>`_ (pictured below). Note that this repo is **private** which means you'll need a team member to add you before you can view it. 
```

Comment:
```suggestion
      - `Sankey diagrams of transitions between states <https://github.com/ihmeuw/vivarium_research_multiple_myeloma/tree/8ca7c6d23354ffb08f532d163990f18745f4c80a/verification/interactive_simulations/sankey_diagrams>`_ (pictured below). Note that this repo is **private** which means you'll need a team member to add you before you can view it. 
```

Is that helpful at all? Not sure

Reply:
Helpful to people at IHME

Reply:
I meant the additional words added; yep, I see how this link could be helpful for someone internal (where it shouldn't be too much of a hassle to get access).